tests/test_build/sidebar_subpage.html

```diff
@@ -1,18 +1,22 @@
 <div class="col-12 col-md-3 bd-sidebar">
- <nav aria-label="Main navigation" class="bd-links" id="bd-docs-nav">
-  <div class="bd-toc-item active">
-   <ul class="nav bd-sidenav">
-    <li class="toctree-l1">
-     <a class="reference internal" href="page1.html">
-      3.1. Section 1 page1
-     </a>
-    </li>
-    <li class="toctree-l1">
-     <a class="reference external" href="https://google.com">
-      https://google.com
-     </a>
-    </li>
-   </ul>
-  </div>
- </nav>
+ <div class="sidebar-start-items">
+  <nav aria-label="Main navigation" class="bd-links" id="bd-docs-nav">
+   <div class="bd-toc-item active">
+    <ul class="nav bd-sidenav">
+     <li class="toctree-l1">
+      <a class="reference internal" href="page1.html">
+       3.1. Section 1 page1
+      </a>
+     </li>
+     <li class="toctree-l1">
+      <a class="reference external" href="https://google.com">
+       https://google.com
+      </a>
+     </li>
+    </ul>
+   </div>
+  </nav>
+ </div>
+ <div class="sidebar-end-items">
+ </div>
```

Comment:
Did you generate these test files automatically somehow or just manual creation accordingly to the structure you are waiting to get?

Reply:
they're automatically created using our pytest-regressions!

Reply:
OK, I now remember it :wink:
Something in the back of my head was telling me this was automatically generated.